dotnet/src/webdriver/Internal/FileUtilities.cs

```diff
@@ -186,12 +186,14 @@ public static string GetCurrentDirectory()
 
             string currentDirectory = location;
 
+#if !NET8_0_OR_GREATER
             // If we're shadow copying, get the directory from the codebase instead
             if (AppDomain.CurrentDomain.ShadowCopyFiles)
             {
                 Uri uri = new Uri(executingAssembly.CodeBase);
                 currentDirectory = Path.GetDirectoryName(uri.LocalPath);
             }
+#endif
```

Comment:
`Assembly.CodeBase` [always throws an exception](https://learn.microsoft.com/en-us/dotnet/api/system.reflection.assembly.codebase?view=net-8.0) in modern .NET.

Luckily, `AppDomain.ShadowCopyFiles` always returns `false` in modern .NET; the concept of shadow copying does not exist anymore.

These changes fix warnings and simplify the code path.

Reply:
We cannot use preprocessor directives. `netstandard2.0` can be consumed by `.net6`. I am not sure what alternatives I can propose, probably just to suppress the warning (only if we cannot find out better way).

Reply:
There's no problem with including this in .NET standard to be consumed by .NET 6. `AppDomain.CurrentDomain.ShadowCopyFiles` will just return `false` like before.

This is just addressing the warning when compiling against .NET 8, produced by `Assembly.CodeBase`.

Reply:
It would be nice if modern .NET recognized at compile time, that anything hidden behind `AppDomain.CurrentDomain.ShadowCopyFiles` is dead code. But that's unfortunately not the case, so this directive is necessary to address warnings